musicname/ze_genso_of_last_v2_2fix.cfg

```
@@ -5,9 +5,9 @@
     "lvl2music_avaterost.mp3"               "James Horner - Becoming one of The People - Becoming one with Neytiri"
     "lvl3music_day_one.mp3"                 "Twelve Titans Music - Day One"
     "lvl3music1_2_eon.mp3"                  "Ninja Tracks - Eon"
-    "lvl3music1_ice Of_phoenix.mp3"         "Audiomachine - Ice of Phoenix"
+    "lvl3music1_ice of_phoenix.mp3"         "Audiomachine - Ice of Phoenix"
     "lvl3music2_redrum.mp3"                 "Immediate Music - Redrum"
-    "lvl4bossMusic_bipolar nightmarel.mp3"  "Keiichi Okabe - Bipolar Nightmare"
+    "lvl4bossmusic_bipolar nightmarel.mp3"  "NieR: Automata OST - Bipolar Nightmare"
```

I personally would go like this isntead
```suggestion
    "lvl4bossmusic_bipolar nightmarel.mp3"  "Keiichi Okabe - Bipolar Nightmare (NieR: Automata OST)"
```